packages/@o3r/test-helpers/tsconfig.spec.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../../tsconfig.jest",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "target": "es2016",
```

Comment:
`target` and `types` node needed (will be follow workspace tsconfig.jest.json to make them evolve with the jest/node versions when upgrading)